docs/index.rst

```diff
@@ -9,9 +9,10 @@ resources targeted for edX learners, educators, researchers, and Open edX operat
     :hidden:
 
     Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
-    Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
+    developers_guide/*
```

Comment:
This generates a warning:
```
edx-developer-docs/docs/index.rst:8: WARNING: toctree contains reference to nonexisting document 'developers_guide/*'
```
You can use `developers_guide/index` if you want an "Open edX Developers' Guide" entry in the sidebar, or add `:glob:` to the options to include all the files with the pattern.

Reply:
Best option: use `developers_guide/index` here, and add this to developers_guide/index.rst:
```

.. toctree::
    
    micro_frontends_in_open_edx
```

Reply:
Good stuff, thank you